src/utils/constants.js

```diff
@@ -1,18 +1,28 @@
 export const DECK_HANDS = ['2', '3', '4', '5', '6', '7', '8', '9', '10', 'J', 'Q', 'K', 'A'];
 export const DECK_SUITS = ['A', 'B', 'C', 'D', 'E', 'T'];
-export const INVALID_CARDS = ['B2', 'T2', 'T3', 'T4'];
 export const TRUMP_CARD_PREFIX = 'T';
 export const DEFAULT_START_SUIT = 'E';
 export const GAMEMODE_EOP = "Elevation of Privilege";
 export const GAMEMODE_CORNUCOPIA = "OWASP Cornucopia";
 export const DEFAULT_GAME_MODE = "Elevation of Privilege";
+export const INVALID_CARDS = {[GAMEMODE_EOP]: ['E2', 'T2', 'T3', 'T4'], [GAMEMODE_CORNUCOPIA]: []};
 export const STARTING_CARD_MAP = {
-  'D': 'D2',
-  'E': 'E3',
-  'C': 'C2',
-  'B': 'B2',
-  'A': 'A2',
-  'T': 'T5'
+  [GAMEMODE_EOP]: {
```

Comment:
This breaks https://github.com/matthewejones/elevation-of-privilege/blob/13423f19533288308a7473aee6a311626565e4e7/src/client/pages/create.js#L215 

As a result the dropdown for choosing a start suit in the create page is empty.  Better check all occurances of `STARTING_CARD_MAP` and `INVALID_CARDS`

Reply:
I just saw this after I posted it. I pushed a commit that should fix this: https://github.com/dehydr8/elevation-of-privilege/pull/138/commits/314a3f560b7524894d72f91910f9d73483a527d5